components/tools/OmeroPy/test/integration/gatewaytest/test_annotation.py

```diff
@@ -308,6 +308,35 @@ def testFileAnnotation(author_testimg_generated, gatewaywrapper):
     assert gateway.getObject("Annotation", annId) is None
 
 
+def testFileAnnotationSpeed(author_testimg_generated, gatewaywrapper):
+    """ Tests speed of loading file annotations. See PR: 4176 """
+    tempFileName = "tempFile"
```

Comment:
This should be replaced by `tempfile.NamedTemporaryFile()` which has auto-delete semantics, will not cause file creation race conditions, etc.